app/streaming/video/ffmpeg-renderers/d3d11va.cpp

```diff
@@ -150,11 +174,130 @@ D3D11VARenderer::~D3D11VARenderer()
     else {
         SAFE_COM_RELEASE(m_Device);
         SAFE_COM_RELEASE(m_DeviceContext);
+        SAFE_COM_RELEASE(m_VideoDevice);
+        SAFE_COM_RELEASE(m_VideoContext);
     }
 
     SAFE_COM_RELEASE(m_Factory);
 }
 
+/**
+ * \brief Set Monitor HDR MetaData information
+ *
+ * Get the Monitor HDT MetaData via LimeLight library
+ *
+ * \param PSS_HDR_METADATA* HDRMetaData The varaible to set the metadata information
+ * \return bool Return True is succeed
+ */
+void D3D11VARenderer::setHDRStream(){
+    DXGI_HDR_METADATA_HDR10 streamHDRMetaData;
+    // Prepare HDR Meta Data for Stream content
+    SS_HDR_METADATA hdrMetadata;
+    // if (m_VideoProcessor.p && m_IsHDRenabled && LiGetHdrMetadata(&hdrMetadata)) {
+    if (m_VideoProcessor.p && LiGetHdrMetadata(&hdrMetadata)) {
```

Comment:
When using GFE instead of Sunshine, with HDR on, the screen will be overbright. Just like it does when using Sunshine+HDR with HDR disabled in moonlight.
[GFE HDR Streaming.zip](https://github.com/moonlight-stream/moonlight-qt/files/14463412/GFE.HDR.Streaming.zip)